internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-nlb-dev.stack.yml

```diff
@@ -502,12 +536,12 @@ Resources: # If a bucket URL is specified, that means the template exists.
     Type: AWS::ElasticLoadBalancingV2::Listener
     Properties:
       DefaultActions:
-        - TargetGroupArn: !Ref NLBTargetGroup1
+        - TargetGroupArn: !Ref NetworkLoadBalancerTargetGroup1
           Type: forward
-      LoadBalancerArn: !Ref PublicNetworkLoadBalancer
+      LoadBalancerArn: !Ref PublicNetworkLoadBalancerV2
       Port: 8082
       Protocol: TCP
-  NLBTargetGroup:
+  NetworkLoadBalancerTargetGroup:
```

Comment:
Why do we want to rename this?

Reply:
The target group can only be attached to one listener rule at a time, so we need to entirely replace them while the listener rule is replaced.